packages/react-router/src/link.tsx

```diff
@@ -275,17 +272,19 @@ type PostProcessParams<
 type ResolveFromParamType<TParamVariant extends ParamVariant> =
   TParamVariant extends 'PATH' ? 'allParams' : 'fullSearchSchema'
 
+type ResolveFromAllParams<
```

Comment:
is this the same as `ResolveAllToParams`?

if yes I would not duplicate this

Reply:
They're different because of `fullSearchSchemaInput/fullSearchSchema`